x/oracle/spec/02_state.md

```diff
@@ -0,0 +1,61 @@
+<!--
+order: 2
+-->
+
+# State
+
+## ExchangeRate
+
+An `sdk.Dec` that stores an exchange rate against USD, which is used by the [Leverage](../../leverage/spec/README.md) module.
+
+- ExchangeRate: `0x01<denom_Bytes> -> amino(sdk.Dec)`
```

Comment:
For cases where a trivial conversion to bytes is done (e.g. `[]byte(string)` or `sdk.Dec.Marshal()`, no protobuf or amino involvement) - it might be simpler to not explicitly denote encoding at all: 

```suggestion
- ExchangeRate: `0x01 | denom -> sdk.Dec`
```
See this example for comparison, though they do it a bit differently than either of us:
https://github.com/cosmos/cosmos-sdk/blob/v0.44.5/x/bank/spec/01_state.md